diagnostic.md

```diff
@@ -149,10 +168,9 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
-
+Div 1 will float to the top right corner. The other divs will float the the left
+and stack on top of each other (according to JS fiddle). I thought the others would
+form a line in the top row but testing showed me otherwise.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```